### PR TITLE
Allow trailing whitespace in the IPAC header lines

### PR DESCRIPTION
**Pull request by taldcroft**
Partial fix for #4749.

### CODE REVIEW

**Comment by pllim** [comment]
See #4749

**Comment by pllim** [comment]
Perhaps add a change log?

**Comment by taldcroft** [comment]
Changelog updated, ready for final review.

**Comment by pllim** [comment]
:+1: I don't see anything controversial here. There is an option to squash on merge now, instead of having to manually squash commits by the author. But as @eteq found out, it doesn't actually squash the PR, just the commit on `master`.

File: CHANGES.rst [hunk 1]
```diff
@@ -29,6 +29,8 @@ New Features
   - Check that columns in ``formats`` specifier exist in the output table
     when writing. [#4508]
 
+  - Allow trailing whitespace in the IPAC header lines. [#4758]
+
```
Comment (hamogu) [inline CHANGES.rst]: Should there be a reference to #4749 here?

**Comment by hamogu** [comment]
I think this is fine.

**Pull request merged**